R/utils-read.R

```diff
@@ -626,15 +626,15 @@ readEstimatedTimeSeries <- function(reportObject, seriesName, timezone, descript
         p <- estimatedPeriods[i,]
         startTime <- p$start
         endTime <- p$end
-        if(inverted){
-          estimatedSubset <- rbind(estimatedSubset, seriesData[['points']][seriesData[['points']][['time']] < startTime | seriesData[['points']][['time']] >= endTime,])
-        } else {
-          estimatedSubset <- rbind(estimatedSubset, seriesData[['points']][seriesData[['points']][['time']] >= startTime & seriesData[['points']][['time']] < endTime,])
-        }
+        estimatedSubset <- rbind(estimatedSubset, seriesData[['points']][seriesData[['points']][['time']] >= startTime & seriesData[['points']][['time']] < endTime,])
       }
     }
     #Replace data with only saved data
-    seriesData[['points']] <- estimatedSubset
+    if(inverted){
+      seriesData[['points']] <- seriesData[['points']][ !(seriesData[['points']][['time']] %in% estimatedSubset[['time']]), ]
```

Comment:
@lindsaycarr I tagged you for a review because I'm wondering if you know of a better way to do this than what I'm doing here (subsetting). I tried dplyr but couldn't get it to work, so I'm wondering if you have any advice.